x-pack/plugins/security_solution/server/endpoint/services/actions/clients/lib/base_response_actions_client.ts

```diff
@@ -73,12 +76,15 @@ export interface ResponseActionsClientUpdateCasesOptions {
   hosts: Array<{
     hostname: string;
     hostId: string;
+    agentType: ResponseActionAgentType;
```

Comment:
can you remove this `agentType` from here?
Adding this property to the `cases` attachment payload should be the responsibility of this method since we already know what that is. Every response action client must defined a static property named `agentType` which we can access via `this.agentType`.  This `updateCases()` method should just add it automatically to each case comment when it is preparing the update to cases

Reply:
Of course, good idea! Thanks!